### DISCUSSION TITLE
Shredded Globular Cluster

### DISCUSSION THREAD

**Discussion by MJoseph315** [a-cosmology-group/acg#101] [Astrophysics and Physics]
Thoughts on this new study: (https://www.sydney.edu.au/news-opinion/news/2020/07/30/stars-that-time-forgot-phoenix-stream-globular-cluster-metallicity.html)?

**Comment by RedshiftDrift** [comment]
Globular clusters are known to contain very old stars (some estimated to be older than 13.8 Gyrs).  However, the mechanism that would lead to the destruction of the globular cluster is not clear because it relies on our limited knowledge of galaxy formation, which in turn is strongly relying on ΛCDM.

The article says
>  The evidence strongly suggests the original structure was the last of its kind, a globular cluster whose birth and life were different to those remaining today.

However there is no evidence, independent of the ΛCDM model, that the birth and life of globular structures were different to those remaining today.

This paper that "confirms ΛCDM" does not really help us in understanding what's wrong with the Big Bang model.
